tests/cases/integration/MigrationsExtension.diffGenerator.configA.neon

```diff
@@ -0,0 +1,10 @@
+migrations:
+	dir: migrations
+	dbal: doctrine
+	driver: Nextras\Migrations\Drivers\MySqlDriver(..., m)
```

Comment:
to tu omylem zůstalo z jiného testu; ničemu to nevadí, ale je to useless, takže to zruším